stable/prometheus-rabbitmq-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 0.2.0
-appVersion: v0.29.0
+version: 0.2.1
+appVersion: v0.29.1
```

Comment:
Don't bump the appVersion. There is no v0.29.1 release: https://github.com/kbudde/rabbitmq_exporter/releases